server.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express');
+const path = require('path');
+const dotenv = require('dotenv');
+
+dotenv.config();
+
+const app = express();
+const port = process.env.PORT || 7000;
+
+app.use(express.static(path.join(__dirname, 'src')));
+
+app.get('*', (req, res, next) => {
+  res.sendFile(path.join(__dirname, 'src/index.html'));
+});
+
+
+app.listen(port, () => {
```

Comment:
I tried to check for an error but it seems the listen function doesn't return any error even if there's one. Hence, since an error could only occur in development environment and wouldn't get to the user, I felt this was something I could skip for now.